src/reducers/filter.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-expressions */
```

Comment:
With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.